editor/plugins/tiles/tile_data_editors.cpp

```diff
@@ -33,6 +33,7 @@
 #include "tile_set_editor.h"
 
 #include "core/math/geometry_2d.h"
+#include "core/math/math_funcs.h"
```

Comment:
```suggestion
```
Already included in `geometry_2d`

Reply:
Thanks! Visual studio frequently acts up

Reply:
Done